python/Ice/optional/.gitignore

```diff
@@ -0,0 +1,2 @@
+*_ice.py
```

Comment:
For the MATLAB demos, I put everything in the matlab/.gitignore file. Do we prefer a gitignore per demo?

Reply:
Ok, let's use a `.gitignore` per demo.